ui/summit-2024/src/pages/Categories/components/NomineeCard.tsx

```diff
@@ -61,7 +66,7 @@ const NomineeCard: React.FC<NomineeCardProps> = ({
           flexShrink: 0,
           borderRadius: "24px",
           border: `1px solid ${
-            selectedNominee === nominee.id || votedNominee
+            (selectedNominee === nominee.id || votedNominee) && showSelection
```

Comment:
If they aren't allowed to vote, `allowToVote=false` as their wallet isn't verified, should be they be able to click cards?

Reply:
According to [this design](https://www.figma.com/design/cRGPl1mAbAsY9MUuJM14xf/Cardano-Ballot-2024?node-id=452-6144&t=5I72gL5oWCUqRzPp-4) from Robert they can click on the the cards